ansys/dpf/core/server_context.py

```diff
@@ -63,7 +69,20 @@ class AvailableServerContexts:
     """Loads the minimum number of plugins for a basic usage. Is the default."""
 
 
+DPF_SERVER_CONTEXT_ENV = "ANSYS_DPF_SERVER_CONTEXT"
+
```

Comment:
this is not related to this PR, maybe you could move it to another one please?

Reply:
See #633